data/concepts.yml

```diff
@@ -116,6 +116,7 @@ toc:
   - docs/concepts/cluster-administration/networking.md
   - docs/concepts/cluster-administration/network-plugins.md
   - docs/concepts/cluster-administration/logging.md
+  - docs/concepts/cluster-administration/monitoring.md
```

Comment:
A new section on monitoring is a great idea.

I recommend:
- move https://kubernetes.io/docs/concepts/cluster-administration/controller-metrics/ into this section
- configure a redirect to the new URI

Also: how about adding an issue to document _workload_ monitoring? It'd be a good change that doesn't have to happen in this PR.

Reply:
How to configure redirect?

Reply:
Used aliases, PTAL if it make sense.